Libraries/CO.CDP.Mvc.Validation/ValidEmailAddressAttribute.cs

```diff
@@ -0,0 +1,27 @@
+using CO.CDP.Localization;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+using System.ComponentModel.DataAnnotations;
+using System.Net.Mail;
+
+namespace CO.CDP.Mvc.Validation;
+
+public class ValidEmailAddressAttribute : ValidationAttribute
+{
+    protected override ValidationResult IsValid(object? value, ValidationContext validationContext)
```

Comment:
Test missing.

Reply:
Should be fine i think

![image](https://github.com/user-attachments/assets/0fa57791-4598-4986-873b-e4f51f64352f)

Reply:
Test coverage provided by already existing, updated tests